examples/video_analysis_per_frame.py

```diff
@@ -30,7 +30,7 @@ def forFrame(frame_number, output_array, output_count, returned_frame):
 
     if (resized == False):
         manager = plt.get_current_fig_manager()
-        manager.resize(width=1000, height=500)
+        manager.resize(1000, 500)
```

Comment:
It looks like newer versions on matplotlib renamed this parameter. According to the [current docs available](https://matplotlib.org/3.1.0/api/backend_bases_api.html#matplotlib.backend_bases.FigureManagerBase.resize), parameters are `w` and `h`. Maybe they were different when @OlafenwaMoses wrote this code. One more reason to fix libraries versions on `requirements.txt`